barretenberg/cpp/src/barretenberg/join_split_example/proofs/join_split/join_split.test.cpp

```diff
@@ -706,10 +706,9 @@ TEST_F(join_split_tests, test_0_input_notes_and_detect_circuit_change)
     EXPECT_TRUE(result.valid);
 
     // The below part detects any changes in the join-split circuit
-
-    constexpr uint32_t CIRCUIT_GATE_COUNT = 184517;
-    constexpr uint32_t GATES_NEXT_POWER_OF_TWO = 524288;
-    const uint256_t VK_HASH("2e1b6e64cd16912f2740d84a0e6c9e01784b35e09b303a026cc58ff5d6a5934d");
+    constexpr uint32_t CIRCUIT_GATE_COUNT = 49492;
```

Comment:
joint split used to be 184K now its 49K, I assume this is mainly because we were using turbo plonk before and now we are using ultraplonk